OWNERS_ALIASES

```diff
@@ -38,7 +33,6 @@ aliases:
   - hongweiliu17
   - Josh-Everett
   - sonam1412
-  - MartinBasti
```

Comment:
I think Martin needs to be added to the build team alias, wdyt?

Reply:
Martin is currently in the Gold team which handles cachi2 things which has no e2e-tests at the moment IIRC. 
I DM'd him and he's fine with not being in the OWNERS for now

Reply:
Oh yeah that makes sense, thanks for checking.